### PR TITLE
Specify Ruby version in Gemfile and update gems

### PR DESCRIPTION
**Pull request by shadyoak**
This should quiet our bot friend.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 source "https://rubygems.org"
+
+ruby "2.7.2"
```
Comment (dperez37) [inline Gemfile:4]: i assume this was the main issue?

Comment (shadyoak) [reply Gemfile:4]: Apparently not. But it did expose the fact that Travis isn't actually honoring the `.ruby-version` file.

Comment (dperez37) [reply Gemfile:4]: Good to know.

**Review by dperez37 — APPROVED** [review]
Bravo!

**Pull request merged**